crates/sui-graphql-rpc/src/metrics.rs

```diff
@@ -62,10 +261,68 @@ impl RequestMetrics {
                 registry,
             )
             .unwrap(),
-            _db_query_cost: register_histogram_with_registry!(
-                "db_query_cost",
-                "Cost of a DB query",
-                DB_QUERY_COST_BUCKETS.to_vec(),
+            query_payload_error: register_int_counter_vec_with_registry!(
+                "query_payload_error",
+                "The total number of client input errors due to too large payload size",
+                &["path"],
+                registry,
+            )
+            .unwrap(),
+            query_validation_latency_by_path: register_histogram_vec_with_registry!(
+                "query_validation_latency_by_path",
+                "The time to validate the query for each path",
+                &["path"],
+                LATENCY_SEC_BUCKETS.to_vec(),
+                registry,
+            )
+            .unwrap(),
+            query_validation_latency: register_histogram_with_registry!(
+                "query_validation_latency",
+                "The time to validate the query",
+                LATENCY_SEC_BUCKETS.to_vec(),
+                registry,
+            )
+            .unwrap(),
+            query_latency: register_histogram_with_registry!(
```

Comment:
Is this inclusive of the validation latency?

Reply:
Yes

Reply:
I actually lost my mind on these, so we do have query validation latency.